test/stdlib/CodableTests.swift

```diff
@@ -1,15 +1,6 @@
-// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
-// Licensed under Apache License v2.0 with Runtime Library Exception
-//
-// See https://swift.org/LICENSE.txt for license information
-// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-//
-//===----------------------------------------------------------------------===//
-//
```

Comment:
Why remove the copyright header?

Reply:
I've restored the copyright header, with the correct years.
(However, the majority of stdlib tests don't have copyright headers.)